AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -488,7 +540,18 @@ - (CGSize)calculatedSizeForNodeAtIndexPath:(NSIndexPath *)indexPath
 
 - (ASCellNode *)nodeForItemAtIndexPath:(NSIndexPath *)indexPath
 {
-  return [_dataController nodeAtIndexPath:indexPath];
+  return [_dataController nodeAtCompletedIndexPath:indexPath];
+}
+
+- (NSIndexPath *)convertIndexPathFromCollectionNode:(NSIndexPath *)indexPath waitingIfNeeded:(BOOL)wait
```

Comment:
This is private, and only here for backwards-compatibility reasons. Currently the `scrollToItemAtIndexPath` methods on `ASTableView` and `ASCollectionView` take index paths in the data source space, and so for the time being we use this as a shim to maintain that behavior. In the future, after some testing, we'll remove our override on that method, and remove this with it.